packages/message-handler/src/abstract-message-handler.ts

```diff
@@ -19,6 +19,7 @@ export abstract class AbstractMessageHandler {
     if (this.nextMessageHandler) {
       return this.nextMessageHandler.handle(message, context)
     }
+    console.warn("Can't handle message: ", message)
```

Comment:
We should be using the `debug` module for this kind of logging, since that can be switched off or filtered.